resume-input/src/Components/Basics/basics.scss

```diff
@@ -0,0 +1,18 @@
+.basic {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding: 5px;
+  box-sizing: border-box;
+  border: 1px solid #c7c7c7;
+  border-radius: 4px;
+
+  &__pairs {

```

Comment:
I have not seen this selector before. I'll have to look it up!

Reply:
It's SASS, so basically if the class is 
```<div class="button"><p class="button__pairs">inside</p></div>```
in SCSS it would be ```.button { &__pair{ some style }}```
It's an easier way to write css
https://sass-lang.com/